cirq/experiments/cross_entropy_benchmarking.py

```diff
@@ -0,0 +1,363 @@
+import numpy as np
+from typing import List, Set, Tuple, Sequence, Dict, Any, NamedTuple, Union, \
+    Iterable
+
+from matplotlib import pyplot as plt
+from cirq import devices, ops, circuits, sim
+
+CrossEntropyPair = NamedTuple('CrossEntropyPair', [('num_cycle', int),
+                                                   ('xeb_fidelity', float)])
+
+
+class CrossEntropyResult:
+    """Results from a cross-entropy benchmarking (XEB) experiment."""
+
+    def __init__(self, cross_entropy_pair: Sequence[CrossEntropyPair]):
+        """
+        Args:
+            cross_entropy_pair: A sequence of NamedTuples, each of which
+                contains two fields: num_cycle which returns the circuit
+                depth as the number of cycles and xeb_fidelity which returns
+                the XEB fidelity after the give cycle number.
+        """
+        self._data = cross_entropy_pair
+
+    @property
+    def data(self) -> Sequence[CrossEntropyPair]:
+        """Returns a sequence of NamedTuples, each of which contains a cycle
+        number and the corresponding XEB fidelity.
+        """
+        return self._data
+
+    def plot(self, **plot_kwargs: Any) -> None:
+        """Plots the average XEB fidelity vs the number of cycles.
+
+        Args:
+            **plot_kwargs: Arguments to be passed to matplotlib.pyplot.plot.
```

Comment:
Backticks (\`) around `matplotlib.pyplot.plot`

Reply:
Done.

Reply:
I think you may have not pushed your changes before saying done.

Reply:
Yes, still working on your last few comments. Will push the changes in a couple of hours.